subcommittee_policy_v3.md

```diff
@@ -302,7 +306,7 @@ Subcommittee Wind Up
   * Trading subcommittees do not keep the accounting system
     up to date.
 
-  * The subcommittee for not hold annual board elections
+  * The subcommittee for not hold annual board elections.
```

Comment:
I'm having trouble understanding what that even _means_ - I don't think the addition of the full-stop helps (or hinders) that.

Reply:
That's what you get  when a muddled mind commits things to paper.  However in my defence it wasn't meant for public review - it was put in Github so the committee could edit it concurrently at the F2F, but in the end we used another method.  Many more eyes have looked over it now, so this version will not be the one committed.